safe/safe.go

```diff
@@ -0,0 +1,21 @@
+package safe
+
+import (
+	"log"
+	"runtime/debug"
+)
+
+// GoSafe starts a recoverable goroutine
+func GoSafe(goroutine func()) {
+	go func() {
+		defer recoverGoroutine()
+		goroutine()
+	}()
+}
+
+func recoverGoroutine() {
```

Comment:
For future uses, I would rename this `defaultRecoverGoroutine` and add a new `GoWithRecover` method (*meh* for the name though :stuck_out_tongue_closed_eyes:) that would allow to execute a custom function for recovering. @emilevauge WDYT ?